test/e2e/tests/confirmations/transactions/speed-up-and-cancel-confirmations.spec.ts

```diff
@@ -89,8 +92,8 @@ describe('Speed Up and Cancel Transaction Tests', function () {
           // Create initial stuck transaction
           await createDappTransaction(driver, {
             value: ethInHexWei(0.1),
-            maxFeePerGas: '0x22ae4b8bcb',
-            maxPriorityFeePerGas: '0x59682f04',
+            maxFeePerGas: decimalToPrefixedHex(1),
+            maxPriorityFeePerGas: decimalToPrefixedHex(1),
```

Comment:
mmm I don't think changing the gas values is needed, as the mining is already stopped until we mine manually, and the race condition is at the exact point between confirming the replacement tx and mining. Wdyt?